### PR TITLE
Ei 776 eups template generation

### PR DESCRIPTION
**Pull request by David-B-Read**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/ManualPayments/IManualPaymentsService.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using OfficeOpenXml;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.ManualPayments.Types;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.ManualPayments
+{
+    public interface IManualPaymentsService
+    {
+        Task<List<PaymentRecord>> LoadPaymentRecords(Stream blobStream);
+        Task<ExcelPackage> LoadManualPaymentsExcelTemplate();
+        Task<ExcelPackage> AddManualPaymentsToTemplate(ExcelPackage excelPackage, List<PaymentRecord> paymentRecords);
```
Comment (Stephek) [inline src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/ManualPayments/IManualPaymentsService.cs:13]: Should this not return a void or Task?

File: src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleManualTemplateFileUpload.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        private readonly IManualPaymentsService _manualPaymentsService;
+        private const int PaymentRecordsLimit = 2500;
+
+        public HandleManualTemplateFileUpload(IManualPaymentsService manualPaymentsService)
+        {
+            _manualPaymentsService = manualPaymentsService;
+        }
+
+        [FunctionName("HandleManualTemplateFileUpload")]
+        public async Task Run([BlobTrigger("employer-incentives-eups/PaymentData/{name}", Connection = "ConfigurationStorageConnectionString")]Stream blobStream, string name, ILogger log)
```
Comment (Stephek) [inline src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleManualTemplateFileUpload.cs:21]: How can we validate what is uploaded?  Any crap could be passed in in any order and we may end up loading invalid data?

Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleManualTemplateFileUpload.cs:21]: If it was a public facing service, I would agree, however it's an internal tool used for an exception process in the event that the payments API is unavailable, and will only be used by members of the team/DevOps in a test environment

File: src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/ManualPayments/IManualPaymentsService.cs (hunk 1 above)
Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/ManualPayments/IManualPaymentsService.cs]: Updated

**Review by Stephek — APPROVED** [review]

**Comment by David-B-Read** [comment]
Closed as functionality is not required

**Pull request closed without merging**